quest/mp2.py

```diff
@@ -10,12 +10,24 @@ def df_mp2(wfn):
     pass
 
 
-def _mo_transform():
+def _mo_transform(orbital_energies):
+    
     pass
 
 
-def _denom():
-    pass
+def _denom(fock_matrix):
+    #get energies from fock matrix)
```

Comment:
I don't think this will need to take a `fock_matrix`, could be a wavefunction, or even just the `eps` array directly.